go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsxrayexporter v0.42.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/azuremonitorexporter v0.42.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/carbonexporter v0.42.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/coralogixexporter v0.42.0
```

Comment:
Don't forget to also add this exporter to the contrib release: https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/manifest.yaml

Reply:
Thanks @jpkrohling ,
https://github.com/open-telemetry/opentelemetry-collector-releases/pull/69